test/drivers/test_suites/syscall_exit_suite/clone_x.cpp

```diff
@@ -223,14 +223,6 @@ TEST(SyscallExit, cloneX_child)
 
 	evt_test->disable_capture();
 
-	/* In some architectures we are not able to catch the `clone exit child
-	 * event` from the `sys_exit` tracepoint. This is because there is no
-	 * default behavior among different architectures... you can find more
-	 * info in `driver/feature_gates.h`.
-	 */
-#ifdef CAPTURE_SCHED_PROC_FORK
```

Comment:
now we enable the tracepoint under the hood when corresponding syscalls are enabled for example:
* `sched_proc_fork` is enabled when one syscall between `clone`/`clone3`/`fork`/`vfork` is enabled